tests/Frameworks/Slim/Version_4/app/routes.php

```diff
@@ -23,4 +25,24 @@
         $group->get('', ListUsersAction::class);
         $group->get('/{id}', ViewUserAction::class);
     });
+
+    $app->get('/simple', function (Request $request, Response $response, $args) {
+        $response->getBody()->write("Simple :)\n");
+        return $response;
+    })->setName('simple-route');
+
+    // Create Twig
+    $twig = Twig::create(__DIR__ . '/../templates'/*, ['cache' => 'path/to/cache']*/);
```

Comment:
Two questions here:

1. Are we supposed to escape or sanitize this directory for information disclosure?
2. Should we keep the commented out cache bit?

Reply:
1. This functions the same as Slim 3: the `slim.view` span has the template file name in the metadata but this directory isn't part of the metadata. Is that what you're referring to?
2. Good catch; removed.

Reply:
1. This is why I took a nap.
2. Thanks!